README.md

```diff
@@ -75,3 +75,12 @@ If you prefer to run a local **development** server directly, you can use instea
 ```sh
 % npm run start
 ```
+
+Supported Browsers
+------------------
+
+This application should run on the latest version of the following browsers:
```

Comment:
Or should I write "This application *runs* on the..."? Depends on how confident we are in our QA :grin:

Reply:
Indeed, this sounds a bit like we aren't sure.
While, of course, we aren't, maybe just use something like:

> The following browsers are officially supported:

Reply:
According to https://browserl.ist/?q=%3E1%25%2C+not+dead%2C+not+op_mini+all%2C+last+2+chrome+version%2C+last+2+firefox+version%2C+last+2+safari+version the following browsers are now selected:

![image](https://user-images.githubusercontent.com/2993/68733079-e49eb280-05d5-11ea-8424-29f1b8915fcb.png)

Reply:
@luniki Which notably excludes mobile Firefox. Thanks for checking this, I will add that browser explicitly.